db/warehouse/migrate/20230428210859_clean_up_external_id_index.rb

```diff
@@ -2,12 +2,17 @@ class CleanUpExternalIdIndex < ActiveRecord::Migration[6.1]
   # remove redundant indexes. These columns already indexed by external_ids_uniq_source_value
   # (source_id, source_type, remote_credential_id)
   def change
-    remove_index :external_ids,
-                 [:source_id, :source_type, :remote_credential_id, :value],
-                 unique: true,
-                 name: 'external_ids_uniq'
```

Comment:
I didn't have this index in my db, and the tests were complaining too: https://github.com/greenriver/hmis-warehouse/actions/runs/4893839444/jobs/8737266921
Not sure why that happened, but I made it conditional.

Reply:
I assumed `index_external_ids_on_source` was created by rails automatically but it must just be cruft that ended up in the schema. Thanks for fixing it!